src/rpc/net.cpp

```diff
@@ -543,8 +543,9 @@ static UniValue setban(const JSONRPCRequest& request)
                 },
     };
     std::string strCommand;
-    if (!request.params[1].isNull())
-        strCommand = request.params[1].get_str();
+    if (!request.m_params["command"].isNull()) {
```

Comment:
I think you could use a function instead of exposing m_params, like
```cpp
std::string strCommand = request.GetString("command");
```

Reply:
I don't see any benefit to doing that...?

Reply:
Less code, supports specifying default value, abstract `UniValue`.

Just a suggestion though.

Reply:
I did think of that when making these changes and opened https://github.com/jgarzik/univalue/pull/63 as a road forward in that regard.

At the end of the day, it is unrelated to this PR though...